approved/ip_xact_sub_block.xml

```diff
@@ -3,18 +3,18 @@
   <spirit:vendor>Origen</spirit:vendor>
   <spirit:library>Origen</spirit:library>
   <spirit:name>d_ip_ana_test_annex_syn</spirit:name>
-  <spirit:version></spirit:version>
+  <spirit:version/>
   <spirit:memoryMaps>
     <spirit:memoryMap>
-      <spirit:name></spirit:name>
+      <spirit:name>atx</spirit:name>
       <spirit:addressBlock>
-        <spirit:name></spirit:name>
-        <spirit:baseAddress>0x0</spirit:baseAddress>
-        <spirit:range>4</spirit:range>
+        <spirit:name>atx</spirit:name>
```

Comment:
@chrishume is this also needed? When the subblock is imported, atx gets initialized as a sub_block within the sub block, so it becomes dut.atx2.atx.atx

removing the name from the spirit fields ie: <spirit:name></spirit:name> initializes the subblock correctly as $dut.atx2 and has access to the mclkdiv register

Reply:
Yes, if you leave the name empty, some tools will fail to process the Spirit XML for UVM RDB conversion.  I'll reproduce the error and add it here.

Reply:
so then it looks like, as discussed, that the MemoryMap should not be imported as a sub_block?
@chrishume @ginty

Reply:
Dunno about that, there's a level of hierarchy there in the XML, so it seems right that it should be represented in Origen too.

I do think its OK if making the roundtrip from Origen -> IP-XACT -> Origen results in this extra level being introduced, I can't imagine an application case that would care about this and this does seem like it may only really be an issue for this test case.